src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -48,12 +64,53 @@ module.exports = function(tag : number) : Fiber {
     input: null,
     output: null,
 
-    handler: function() {},
-    handlerTag: 0,
+    stage: 0,
 
     hasPendingChanges: true,
 
-    stateNode: {},
+    stateNode: null,
 
   };
 };
+
+function shouldConstruct(Component) {
+  return !!(Component.prototype && Component.prototype.isReactComponent);
+}
+
+exports.createFiberFromElement = function(element : ReactElement) {
+  const fiber = exports.createFiberFromElementType(element.type);
+  if (typeof element.type === 'object') {
+    // Hacky McHack
+    element = ReactElement(fiber.input, null, element.ref, null, null, null, element.props);
```

Comment:
Never mind, I get it now. `element.type` comes in as object, so `fiber` is just `element.type`, and we apply the props to the continuation type.